### ISSUE TITLE
M22-Paste REP from clipboard - improve the messages on dialog

### ISSUE THREAD

**Issue by helenayele** [debrief/debrief#3254]
We need to change the title of https://gyazo.com/03fb404ccb8653d1d74dc0c29eab94ed dialog. And also it is better to add some clarification what is possible to paste in the description part of this dialog.
### Steps to reproduce this dialog
1. Copy BARRIER CHARLIE from Misc layer in sample.dpf file
2. Open a plot with a track data, open boat1.rep file , Right click on a track, then you will get paste BARRIER CHARLIE option, select it

### Expected Result
Dialog should have meaningful title and also need to provide some information as what is possible to paste in this layer.

**Comment by helenayele** [comment]
Currently the dialog is not shown instead "Exception caught:Can't paste Line:BARRIER CHARLIE into track" info message is logged in debrief  
Video - https://gyazo.com/34da4ca08cbb5d2b6c986df20b09e833

**Comment by ayeshare** [comment]
Yes, I see the message in the error log now.  Has this been fixed?

**Comment by IanMayo** [comment]
That was a workaround, to be honest. Using our new message dialog would be better.
